.travis.yml

```diff
@@ -15,10 +15,10 @@ addons:
       - gdb
 
 script:
-  - export AWS_KVS_LOG_LEVEL=3
+  - export AWS_KVS_LOG_LEVEL=2
   - make
   - ulimit -c unlimited -S
-  - timeout --signal=SIGABRT 60m ./tst/webrtc_client_test
+  - timeout --signal=SIGABRT 60m ./tst/webrtc_client_test --gtest_filter="PeerConnectionFunctionalityTest.*"
```

Comment:
Oh, I see, I think this is for testing purposes and you are going to revert this right/